docs/user/automation-rules.rst

```diff
@@ -82,7 +82,7 @@ Make version private
 Set version as default
   Sets the version as the :term:`default version`.
   It also activates and builds the version.
-  See :ref:`automatic-redirects:Root URL`.
+  See :ref:`root_url_redirect`.
```

Comment:
This is probably something we should have in the style guide as well. Should we use the titles, which are more likely to change, or make manual refs?

Basically: when do we make manual refs for something?

Reply:
If the headlines seems likely to change, we should use labels... and maybe even add link texts if we think the headline is fragile to include in a sentence structure. There are many places where we use the headline in the sentence structure, so it will render an invalid sentence when the headline changes "Read more in our <Guide to custom domains>" will not look nice if it becomes "Read more in our <Custom domains>".

Reply:
The best way to reference is of course to NOT use the headline in the sentence structure.

Reply:
See also: `Custom domains`
See also: `Guide to custom domains`

This is not fragile to headline changes.

Reply:
(the above text we're commenting on is fine though)

Reply:
Will treat this in https://github.com/readthedocs/readthedocs.org/issues/9747